wagn/features/step_definitions/email_steps.rb

```diff
@@ -95,11 +95,15 @@ def current_email_address
   expect(current_email).to have_subject(Regexp.new(text))
 end
 
-Then /^(?:I|they) should see [\|"](.*)[\|"] in the email body$/ do |text|
+Then /^(?:I|they) should see \|(.*)\| in the email body$/ do |text|
   expect(current_email.text_part.body.raw_source).to include("#{text}")
 end
 
-Then /^(?:I|they) should see \/([^"]*?)\/ in the email body$/ do |text|
+Then /^(?:I|they) should see "(.*)" in the email body$/ do |text|
+  expect(current_email.text_part.body.raw_source).to include("#{text}")
+end
+
```

Comment:
Ah, I think I get what you did here now.  Do you think we need all these variants?  If you have a stylistic preference (and we don't use them all in the tests), feel free to pick one.